src/components/TrackersManager.vue

```diff
@@ -0,0 +1,309 @@
+  selectedStreams,
+  filterString,
+  isFilingTrackers,
+  alreadyFiledTrackers,
+  untrackableAffects,
+  isLoadingTrackers,
+} = useTrackers(props.flawId, affects);
+
+const availableAffectStreams = computed(() => {
+  const referenceAffect = props.affects;
+  return unselectedStreams?.value.filter((tracker) => {
+    return referenceAffect.some((refAffect) => {
+      return refAffect.ps_component === tracker.ps_component
+      && refAffect.ps_module === tracker.ps_module;
+    }
+    );
+  });
+});
+
+const selectedAffectStreams = computed(() => {
+  const referenceAffect = props.affects;
+  return selectedStreams?.value.filter((tracker) => {
+    return referenceAffect.some((refAffect) => {
+      return refAffect.ps_component === tracker.ps_component
+      && refAffect.ps_module === tracker.ps_module;
+    }
+    );
+  });
+});
+
+const emit = defineEmits<{
+  'affects-trackers:refresh': [];
+}>();
+
+function handleFileTrackers() {
+  fileTrackers().then(() => {
+    emit('affects-trackers:refresh');
+  });
+}
+</script>
+
+<template>
+  <div class="trackers-manager py-3 px-4" :class="{'embedded': mode === 'embedded'}">
+    <h4 v-if="mode === 'embedded'" class="mb-2 d-flex">
+      Trackers Manager
+    </h4>
+    <div class="d-flex gap-2">
+      <input
+        v-model="filterString"
+        type="text"
+        class="form-control form-control-sm"
+        style="width: 35%;"
+        placeholder="Filter by stream or component name"
+      />
+      <button
+        type="button"
+        :disabled="isLoadingTrackers || availableAffectStreams?.length === 0"
+        class="btn btn-sm btn-black"
+        @click="toggleTrackerSelections(availableAffectStreams)"
+      >
+        <i class="bi bi-check-all" />
+        Select All
+        {{ filterString ? 'Filtered' : '' }}
+      </button>
+      <button
+        type="button"
+        :disabled="isLoadingTrackers || selectedAffectStreams?.length === 0"
+        class="btn btn-sm btn-black"
+        @click="toggleTrackerSelections(selectedAffectStreams)"
+      >
+        <i class="bi bi-x-square" />
+        Deselect All
+        {{ filterString ? 'Filtered' : '' }}
+      </button>
+    </div>
+    <div class="row mt-3">
+      <div class="col">
+        <h5 class="d-inline-block fs-5">Available Trackers</h5>
+        <caption>
+          <span class="fst-italic">
+            (<i class="bi bi-box-arrow-in-right" /> indicates a Suggested Tracker)
+          </span>
+        </caption>
+      </div>
+      <div class="col">
+        <h6 class="fs-5">Selected To File</h6>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col">
+        <div class="mb-2">
+          <div v-if="isLoadingTrackers" class="ms-2 mb-1">
+            <span
+              class="spinner-border spinner-border-sm"
+              role="status"
+            >
+              <span class="visually-hidden">Loading...</span>
+            </span>
+            <span> Loading trackers&hellip; </span>
```

Comment:
why the spaces around it?
```suggestion
            <span>Loading trackers&hellip;</span>
```

I noticed you have them on all the `Loading xxx` and `No available xxx` strings, was that intended?